### DISCUSSION TITLE
Build in Actions failing due to missing bzip2 includes

### DISCUSSION THREAD

**Discussion by alexm98** [Homebrew/discussions#3672] [Writing Formulae/Casks]
My formula fails when building in GitHub actions due to `bzlib` not being present during configure:

```
checking whether to use bzlib... "yes"
checking for location of bzlib includes... /opt/homebrew/opt/bzip2/include
configure: error: *** bzlib includes directory /opt/homebrew/opt/bzip2/include does not exist.
```

The dependencies I added regarding to this  are:

```
depends_on "lbzip2"
uses_from_macos "bzip2"
```

There must be something that I'm missing. Shouldn't `depends_on "lbzip2"` be sufficient for this not to happen?

Here is the [URL to the actions](https://github.com/Homebrew/homebrew-core/actions/runs/3098749374/jobs/5017145107)

**Comment by SMillerDev** [comment]
lbzip will be in the lbzip directory, bzip from in another directory and bzip from macOS in the command line tools directory.

**Comment by carlocab** [comment]
> There must be something that I'm missing. Shouldn't `depends_on "lbzip2"` be sufficient for this not to happen?

Maybe, but you're telling `configure` to look in a path that does not exist [here](https://github.com/Homebrew/homebrew-core/blob/e8e7125f2e1c1ad86a2eaf0cd412d2ee1e074699/Formula/grass.rb#L97):
```ruby
      "--with-bzlib-includes=#{Formula["bzip2"].opt_include}",
```
